client/src/main/java/edu/colostate/cs/cs414/chesshireCoders/jungleClient/game/JungleGame.java

```diff
@@ -44,44 +49,81 @@ public boolean canMovePieceAt(int row, int column) {
         return result;
     }
 
-    public boolean startGame() {
-        return true;
+    public void endGame() {
+

```

Comment:
Why is there an empty function here?

Reply:
That will be filled when quit is implemented.